src/mbgl/gl/context.cpp

```diff
@@ -98,7 +98,7 @@ void Context::initializeExtensions(const std::function<gl::ProcAddress(const cha
         // Block Adreno 4xx as it crashes in a driver when VBOs are destructed (Android 5.1.1)
         // Block ARM Mali-T720 (in some MT8163 chipsets) as it crashes on glBindVertexArray
         // Block ANGLE on Direct3D as the combination of Qt + Windows + ANGLE leads to crashes
-        if (renderer.find("Adreno (TM) 2") == std::string::npos &&
+        /*if (renderer.find("Adreno (TM) 2") == std::string::npos &&
```

Comment:
Why comment this instead of removing it?

Reply:
Should be removed, not sure why remained commented here. I will fix it now.